webdriver/tests/bidi/input/perform_actions/pointer_pen.py

```diff
@@ -76,7 +76,7 @@ async def test_pen_pointer_properties(
     (
         actions.add_pointer(pointer_type="pen")
         .pointer_move(x=0, y=0, origin=get_element_origin(pointerArea))
-        .pointer_down(button=0, pressure=0.36, tilt_x=-72, tilt_y=9, twist=86)
+        .pointer_down(button=0, pressure=0.36, altitude_angle=0.3137708208317187, azimuth_angle=3.090175787472892, twist=86)
```

Comment:
For testing, I assume values like `1.0` and `2.0` would work as well, right? I don't see a reason why we need all the digits here. This also applies to the tests below.

Reply:
So `tiltX` and `tiltY` are [required to be `long`s](https://w3c.github.io/pointerevents/#dom-pointereventinit-tiltx), so we need to use floats for the altitude and azimuth.